File: Java/troco.java
import java.util.Locale;
import java.util.Scanner;

public class troco {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        Locale.setDefault(Locale.US);

        System.out.print("Preço unitário do produto: ");
        double preco = sc.nextDouble();
        System.out.print("Quantidade comprada: ");
        int quantidade = sc.nextInt();
        System.out.print("Dinheiro recebido: ");
        double dinheiro = sc.nextDouble();

        double troco = Math.abs(dinheiro - preco * quantidade);
        System.out.printf("Troco = %.2f", troco);

        sc.close();

    }
}

File: C/troco.c
#include <stdio.h>

double preco, quantidade, dinheiro, troco;

int main() {
    printf("Preco unitario do produto: ");
    scanf("%lf", &preco);
    printf("Quantidade comprada: ");
    scanf("%lf", &quantidade);
    printf("Dinheiro recebido: ");
    scanf("%lf", &dinheiro);

    troco = dinheiro - preco * quantidade;
    printf("TROCO = %.2lf", troco);
}
